src/backend/web/authentication.js

```diff
@@ -82,8 +82,16 @@ function init(passport) {
 
   // Add session user object de/serialize functions
   passport.serializeUser(function(user, done) {
-    // TODO: is it necessary to use JSON vs. Object?
-    done(null, JSON.stringify(user));
+    const user2 = {
+      name: user['http://schemas.microsoft.com/identity/claims/displayname']
+        ? user['http://schemas.microsoft.com/identity/claims/displayname']
+        : 'Hans Lippershey',
```

Comment:
Hans Lippershay? :smile:  nice touch!
We could also consider saying `Telescope Developer`.